Makefile.ci

```diff
@@ -135,6 +137,8 @@ ci-finmap: ci-mathcomp_1
 ci-bigenough: ci-mathcomp_1
 ci-analysis: ci-elpi_hb ci-finmap ci-bigenough
 
+ci-hb: ci-elpi
+ci-elpi_hb: ci-elpi ci-hb
```

Comment:
```suggestion
ci-elpi_hb: ci-elpi ci-hb
	true
```
Or remove elpi_hb and split the gitlab job

Or remove elpi_hb, rename the gitlab job to just ci-hb and let it implicitly build elpi